RELEASE_NOTES.rst

```diff
@@ -17,32 +17,32 @@ coala 0.8.0 - grizzly
 
 ::
```

Comment:
also in commit shortlog please enclose "writer" and "author" in double backticks


and add a period at the end of the commit body.